scripts/acceptance-cluster-setup-several-k8s-versions.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+# Copyright © 2021 - 2023 SUSE LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+NETWORK_NAME=epinio-acceptance
+MIRROR_NAME=epinio-acceptance-registry-mirror
+CLUSTER_NAME=epinio-acceptance
+export KUBECONFIG=$SCRIPT_DIR/../tmp/acceptance-kubeconfig
+
+# k3s version selection (latest is default)
```

Comment:
Much better now, but I'd still store all the versions like:
```
K3S_RELEASES=$(curl -sH "Accept: application/vnd.github.v3+json" 'https://api.github.com/repos/k3s-io/k3s/releases' | jq -r '.[] | select (.assets[].name == "k3s") | .name')
```
and then you can do the version matching by using this variable inside if/elif conditions.

Also I'd use generic variable `$K3S_IMAGE_TAG` for storing the selected (oldest/latest) version without introducing `$K3S_IMAGE_LATEST/OLDEST` variable and then you can move the `$K3S_IMAGE` outside of the if statement like:
```
...
fi
K3S_IMAGE=${K3S_IMAGE:-rancher/k3s:$K3S_IMAGE_TAG}
echo "Selected k3s image: $K3S_IMAGE"
```
This will safe few lines of code.